### ISSUE TITLE
Remake order is not remaking the order - misleading behavior 

### ISSUE THREAD

**Issue by filrak** [DivanteLtd/vue-storefront#2560]
Currently the 'Remake Order' button in profile is only adding the same products to the cart. It's misleading. At bare minimum it should also redirect to checkout.

**Comment by pkarw** [comment]
@anqaka how are things with this one? :)

**Comment by alinadivante** [comment]
@MikeSheward 
If I use 'Remake order' from dropdown list 
![image](https://user-images.githubusercontent.com/36628922/58459143-fb864100-812a-11e9-8338-2bec39020cb6.png) - it works (I'm redirected to checkout) - **BUT** if I will remove products from cart and I will try again remake the same order then it doesn't work. Im' redirected to Home page and "Shopping cart is empty. Please add some products before entering Checkout" pop-up occurs and at the same time product is added to cart. 
![image](https://user-images.githubusercontent.com/36628922/58459195-1e185a00-812b-11e9-8fe2-1ac40ce4c1ad.png)

What more? If I click on "view order"  from dropdown list and after that I click 'Remake order' link (on order details page) it causes only adding product to cart (without redirecting to checkout) 
![image](https://user-images.githubusercontent.com/36628922/58459251-38523800-812b-11e9-9fea-17c2ac901703.png)

**Comment by pkarw** [comment]
Probably fixed in the https://github.com/DivanteLtd/vue-storefront/pull/2887

**Comment by GabiDivante** [comment]
@pkarw I've been testing this issue on the newest version of your branch - problems reported by Alina no longer occur :) 
But we have got one new issue now - 'remake order' doubles the product quantity in the order:
- sometimes it is visible in the checkout summary
![Peek 2019-06-05 14-22](https://user-images.githubusercontent.com/49188852/58956632-1302a100-879f-11e9-8043-8cbc9fbe3566.gif)

- but often it is visible only after going to the order details of recently placed order
![Peek 2019-06-05 14-18](https://user-images.githubusercontent.com/49188852/58956634-1564fb00-879f-11e9-96cd-efb9ee7b363a.gif)

**Comment by pkarw** [comment]
Could You please re-test? I've been working all day Today on this PR so it might been some side effects of the other fixes. Currently I can't reproduce this error

**Comment by alinadivante** [comment]
Retested, Works for me :) thanks
